LucidGloves/src/Bones.cpp

```diff
@@ -167,40 +167,61 @@ vr::HmdVector4_t CalculatePosition(const float transform, const int boneIndex, c
 	return result;
 }
 
-void ComputeBoneTransform(vr::VRBoneTransform_t bone_transform[NUM_BONES], const float transform, const int startBoneIndex, const bool isRightHand) {
+//Transform should be between 0-1
+void ComputeBoneFlexion(vr::VRBoneTransform_t *bone_transform, const float transform, int i, const bool isRightHand) {
```

Comment:
Can we have the same argument names as what is in the header?

Reply:
Sure